shared/profile/search/container.js

```diff
@@ -1,21 +1,45 @@
 // @flow
 import Search from '.'
 import {createShowUserProfile} from '../../actions/profile-gen'
-import {connect} from '../../util/container'
+import {connect, type RouteProps} from '../../util/container'
 
-type OwnProps = {|onClose: () => void|}
+// Either onClose is passed in from bar.js or we're instantiated via a
+// route.
+type OwnProps = {|onClose: () => void|} | RouteProps<{}, {}>
```

Comment:
@cjb Ah, I see what you mean here. I only need it for `ownProps.navigateUp`, but maybe I should just use `dispatch(RouteTreeGen.createNavigateUp())`?

Reply:
yup!

Reply:
Fixed!